rollup.config.js

```diff
@@ -23,6 +23,8 @@ import ts from 'rollup-plugin-ts'
 
 const production = !process.env.ROLLUP_WATCH
 
+const compilationTimestamp = new Date().getTime()
```

Comment:
can we have a switch here to exclude sourcemaps in production

Reply:
Yes, I'll add a switch. But should sourceMaps be removed from prod? Is this a good practice? (I ran Google Lighthouse which told me that would be an improvement, that's why I made it a default thing).

Reply:
done